saleor/core/utils/url.py

```diff
@@ -33,3 +25,16 @@ def prepare_url(params: str, redirect_url: str) -> str:
     split_url = urlsplit(redirect_url)
     split_url = split_url._replace(query=params)
     return split_url.geturl()
+
+
+def validate_url(url):
+    try:
+        parsed_url = urlparse(url)
+        domain, _ = split_domain_port(parsed_url.netloc)
+        if not parsed_url.netloc:
+            raise ValidationError(
+                "Invalid URL. Please check if URL is in RFC 1808 format."
+            )
+    except ValueError as error:
+        raise ValidationError(error)
+    return domain
```

Comment:
Personally, it is strange that a function with the name  `validate_url` returns the domain.  I wouldn't expect that the validation will return me a part of url.

Reply:
I don't think that this method does the same job as URLValidation. I understand that the issue was TLD, but why we didn't extend the URLValidator to handle the expected cases? Now, from a fully validated URL, we moved to parse URL. 
I would start with something like this:
```
valdiator = URLValidator()
validator.tld_re = "your regex"
valdiator.host_re = '(' + hostname_re + domain_re + tld_re + '|localhost)'
```

Reply:
Good point, I will change that.